broker/config/settings.yml

```diff
@@ -190,11 +190,23 @@ defaults: &defaults
   # APISERVER SETTINGS #
   ##############################
   apiserver:
-    ip: '127.0.0.1'
-    port: 9443
+    ip: '10.244.14.252'
```

Comment:
Shouldn't it remain `localhost` because this file is typically used for local development?

Reply:
running apiserver locally is a challenge as it does not have Mac support. We have come up with a [manifest bosh lite for apiserver](https://github.com/cloudfoundry-incubator/service-fabrik-boshrelease/blob/46ec64d00e618bdba515ac63a047ae9ab44f2a58/templates/apiserver.yml) which sets up everything like etcd, apiserver n all for you. The IP used there is the same as here. We'll come up with a guide to discuss more on that.